nemo/collections/asr/models/configs/matchboxnet_config.py

```diff
@@ -217,6 +222,9 @@ def _finalize_cfg(self):
         # propagate num classes
         self.model_cfg.decoder.num_classes = len(self.model_cfg.labels)
 
+        # propagate task type (regression or classification)
+        self.model_cfg.is_regression_task = self.model_cfg.is_regression_task
```

Comment:
For now, lets revert this, I will update this once the PR merges, theres a little issue with the current approach that I need to think about.